openslides/urls.py

```diff
@@ -31,7 +31,7 @@
     (r'^agenda/', include('openslides.agenda.urls')),
     (r'^motion/', include('openslides.motion.urls')),
     (r'^assignment/', include('openslides.assignment.urls')),
-    (r'^user/', include('openslides.users.urls')),
+    (r'^users/', include('openslides.users.urls')),
```

Comment:
See: https://github.com/OpenSlides/OpenSlides/pull/1459/files#diff-c177be83dbcf178e7067de2c149d71caR14

After the merge you can use /api/users/user/print/